server/api/crud/artifacts.py

```diff
@@ -329,7 +329,7 @@ def _delete_artifact_data(
                 path
             ):
                 raise mlrun.errors.MLRunNotImplementedServerError(
-                    "Deleting artifact data of kind 'dataset' is currently supported for a single file only"
+                    "You can only delete the data of an artifact of kind 'dataset' if it is a single file"
```

Comment:
"Deleting artifact data of kind 'dataset' is currently supported for a single file only"